identity-service/src/notifications/index.js

```diff
@@ -1221,19 +1226,39 @@ class NotificationProcessor {
         emailParams['subject'] = 'Unread notifications from last week'
       }
 
-      console.log(emailParams)
       await this.sendEmail(emailParams)
+
       // Temporary debugging email
       let emailParams2 = emailParams
       emailParams2['to'] = 'hareesh@audius.co'
+      emailParams2['bcc'] = 'forrest@audius.co'
       await this.sendEmail(emailParams2)
+
+      // Cache on file system
+      await this.cacheEmail({ renderProps, emailParams })
+
       return true
     } catch (e) {
       console.log(`Error in renderAndSendEmail ${e}`)
       return false
     }
   }
 
+  async cacheEmail (cacheParams) {
+    let uuid = uuidv4()
+    let timestamp = moment().valueOf()
+    let fileName = `${uuid}-${timestamp.toString()}.json`
```

Comment:
@hareeshnagaraj is this enough info to associate file name to user and params?

Reply:
i mean its all in the file itself, so we can always grep if we need to right?